### PR TITLE
[ADD] open_academy: Add Spanish translation T#59081

### PR DESCRIPTION
**Pull request by JesusValdez96**
- Add es_MX.po file in i18n folder to translate open academy module.
es_mx

Link to exercise: https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#internationalization

### CODE REVIEW

File: open_academy/i18n/es_MX.po [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__duration
+msgid "Duration"
+msgstr "Duración"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.session_view_kanban
+msgid "Duration:"
+msgstr "Duración"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:126]:
```suggestion
msgstr "Duración:"
```

File: open_academy/i18n/es_MX.po [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+#. module: open_academy
+#: model:ir.model.fields.selection,name:open_academy.selection__res_partner__teacher_level__2
+msgid "Level 2"
+msgstr "Nivel 2"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.course_view_search
+msgid "My Courses"
+msgstr "Mis cursos"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:195]:
```suggestion
msgstr "Mis Cursos"
```

File: open_academy/i18n/es_MX.po [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__name
+#: model:ir.model.fields,field_description:open_academy.field_session__name
+msgid "Name"
+msgstr "Nombre"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__number_of_seats
+msgid "Number Of Seats"
+msgstr "Número de asientos"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:206]:
```suggestion
msgstr "Número De Asientos"
```

File: open_academy/i18n/es_MX.po [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.session_view_graph
+msgid "Number of attendees by Course"
+msgstr "Número de asistentes por curso"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:211]:
```suggestion
msgstr "Número de asistentes por Curso"
```

File: open_academy/i18n/es_MX.po [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+#. module: open_academy
+#: model:res.groups,name:open_academy.open_academy_group_manager
+msgid "OpenAcademy / Manager"
+msgstr "OpenAcademy / Administrador"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__responsible_user_id
+#: model_terms:ir.ui.view,arch_db:open_academy.course_view_search
+msgid "Responsible User"
+msgstr "Usuario responsable"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:244]:
```suggestion
msgstr "Usuario Responsable"
```

File: open_academy/i18n/es_MX.po [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+#: model_terms:ir.ui.view,arch_db:open_academy.course_view_form
+#: model_terms:ir.ui.view,arch_db:open_academy.session_view_calendar
+#: model_terms:ir.ui.view,arch_db:open_academy.view_partner_form
+msgid "Sessions"
+msgstr "Sesiones"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__start_date
+msgid "Start Date"
+msgstr "Fecha de inicio"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:271]:
```suggestion
msgstr "Fecha de Inicio"
```

File: open_academy/i18n/es_MX.po [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.session_view_kanban
+msgid "Start Date:"
+msgstr "Fecha de inicio:"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__taken_seats
+msgid "Taken Seats"
+msgstr "Asientos ocupados"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:281]:
```suggestion
msgstr "Asientos Ocupados"
```

File: open_academy/i18n/es_MX.po [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_res_partner__teacher_level
+#: model:ir.model.fields,field_description:open_academy.field_res_users__teacher_level
+msgid "Teacher Level"
+msgstr "Nivel de maestro"
```
Comment (andreaarce) [inline open_academy/i18n/es_MX.po:287]:
```suggestion
msgstr "Nivel del Maestro"
```

**Review by andreaarce — COMMENTED** [review]
Remember that the msgtr should be the same as the msg ID at uppercase characters, characters like " : ", etc. Also, why are you adding a pot file without translations? , if is for English terms the file is not necessary just leave the es_MR.

**Comment by JesusValdez96** [comment]
Changes made @andreaarce, @CarmenMiranda

File: open_academy/i18n/es_MX.po [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__attendee_ids
+#: model_terms:ir.ui.view,arch_db:open_academy.session_view_form
+msgid "Attendees"
+msgstr "Asistentes"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__attendees_count
+msgid "Attendees Count"
+msgstr "Número de asistentes"
```
Comment (CarmenMiranda) [inline open_academy/i18n/es_MX.po:47]:
```suggestion
msgstr "Número de Asistentes"
```

**Comment by CarmenMiranda** [comment]
Even you can name the file `es.po` and it will work for any Spanish language

**Review by andreaarce — APPROVED** [review]

**Review by CarmenMiranda — APPROVED** [review]
@andreaarce Technically LGTM 👍🏼

**Pull request merged**